third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/call_mutual_borrows_invalid.move

```diff
@@ -1,5 +1,5 @@
 module 0x8675309::M {
-    struct S { f: u64, g: u64 }
+    struct S has drop, copy { f: u64, g: u64 }
```

Comment:
Why are we modifying these test inputs?  If it's necessary to exhibit the reference-safety checks, we should also include the original somewhere so we can keep track of differences between V1 and V2 on the same inputs.  I've filed an issue https://github.com/aptos-labs/aptos-core/issues/12712 to go back and re-check these.

Reply:
Please please please, and again, read the PR descriptions.

"Some tests needed to be fixed for this to pass ability checks. "